K8s content restructure (#1217)
fermyon/developer: content/spin/v2/spinkube.md

Reviewer: Suggest keeping the `kubernetes.md` file/page name - there seems little point changing the file name and redirecting given that the title is still "Spin on Kubernetes."

> title = "SpinKube: Spin on Kubernetes"

Reviewer: suggest 

```suggestion
title = "Spin on Kubernetes"
```

because this is kind of a roadmap to SpinKube, FP4K, the old container way, etc. - it's a great landing page and where we want people to come in

Reviewer 2: Thanks @itowlson 
`Spin on Kubernetes` it is.

template = "spin_main"
date = "2024-03-07T00:00:01Z"
enable_shortcodes = true
[extra]
url = "https://github.com/fermyon/developer/blob/main/content/spin/v2/spinkube.md"

---

## Why Use Spin With Kubernetes? 

In addition to `spin up` Fermyon also offers Fermyon Cloud to deploy spin apps into production, so why use Spin with Kubernetes? For users that have made existing investments into Kubernetes or have requirements that their applications stay within certain clouds, not be on shared infrastructure, or run on-premise, Kubernetes provides a robust solution.

There are a few options for running Spin on Kubernetes: 

> *  **SpinKube** is an OSS project that provides the best, Kubernetes-native experience for running Spin applications on Kubernetes. From runtime installation via runtime-class-manager to resource management via Spin Operator, SpinKube provides you with a complete toolkit for running spin applications on Kubernetes as custom resource (known as SpinApps). For guidance on how to get started with SpinKube, please visit the [SpinKube documentation](https://spinkube.dev). 

Reviewer: ```suggestion
*  **SpinKube** is an open source project that provides the best Kubernetes-native experience for running Spin applications on Kubernetes. From runtime installation via `runtime-class-manager` to resource management via Spin Operator, SpinKube provides you with a complete toolkit for running spin applications on Kubernetes as custom resource (known as SpinApps). For guidance on how to get started with SpinKube, please visit the [SpinKube documentation](https://spinkube.dev). 
```

(though I am not quite sure of the grammar with that comma)

Reviewer 2: Thanks @itowlson

  >> If you've been following Fermyon for a while, you may have tried our [legacy Spin integration with Kubernetes], where you bundled Spin applications as OCI containers and ran them in a pod. While a convenient set up for testing, this format introduces limitations such as the need to keep a long running container. For the best experience, we recommend you migrate to [SpinKube](https://spinkube.dev)

> * **Fermyon Platform for Kubernetes** is a managed distribution of SpinKube, currently in private beta, that can be run on your existing Kubernetes infrastructure for enhanced performance and density for your SpinApps. If this of interest to your team, please [schedule a demo](TODO://link-to-demo).

Reviewer 3: `TODO://link-to-demo` to `www.fermyon.com/demo`

> For inspiration on what to build, find answers to your questions, or simply engage with the SpinKube community, check out the [SpinKube Slack channel](TODO:://link-to-demo). 

Reviewer 3: TODO:://link-to-demo > point to [www.fermyon.com/demo](www.fermyon.com/demo) to register interest

Reviewer: This should be the Slack channel rather than the demo

title = "SpinKube"
template = "spin_main"
date = "2023-11-04T00:00:01Z"
enable_shortcodes = true
[extra]
url = "https://github.com/fermyon/developer/blob/main/content/spin/v2/kubernetes.md"

---

> ## Why Use Spin with Kubernetes? 

Reviewer 2: ```suggestion
## Why Use Spin With Kubernetes?
```
